analyzer/Dockerfile

```diff
@@ -0,0 +1,31 @@
+ARG RELEASE=bullseye
+FROM python:3-${RELEASE}
+ARG RELEASE
+
+RUN apt-get update -qq && apt-get upgrade -y -qq
+ADD https://raw.githubusercontent.com/LibreTime/libretime/master/tools/packages.py /tmp/packages.py
+COPY packages.ini /tmp/
+RUN apt-get install -y -qq $(python3 /tmp/packages.py --format=line ${RELEASE} /tmp/packages.ini)
+RUN pip3 install -U pip
+RUN mkdir /var/log/airtime/
+RUN ln -s /dev/stdout /var/log/airtime/airtime_analyzer.log
+
+WORKDIR /app
+
+COPY setup.py ./
+# It will fail because the source code is not installed yet.
+# Only required for the requirements file, which is written
+# before the failure. This is broken up to prevent needing
+# to reinstall all dependencies when the source code changes
+# and the Docker image is rebuilt
+RUN python setup.py egg_info || true
+# seup.py cannot filter out extra_requires, so remove the
+# [dev] section and leave only the install_requires section
+RUN sed -i '/\[dev\]/,$d' libretime_analyzer.egg-info/requires.txt
```

Comment:
Have you tried to build a wheel in a build stage and only import / install the wheel ? Maybe this will be cleaner.

https://github.com/jooola/debouncer/blob/2e87c6b51822915176ff1738b978680104b38ea5/Dockerfile#L6-L20

Reply:
Not exactly cleaner, but reduces the image size slightly